lib/syscall/syscall_x86.c

```diff
@@ -172,6 +172,10 @@ long sys_get_device_info(user_addr_t info)
 		dev_info->attkb_size = 0;
 #endif
 	}
+	else {
+		memset(dev_info->sec_info.attkb_enc_key, 0,
```

Comment:
by the way, why did you define "secure_memzero", but not use it in your patch? did i miss something?